go/server/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"github.com/feast-dev/feast/go/feast"
+	"github.com/feast-dev/feast/go/protos/feast/serving"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+	"os"
+	"strings"
+)
+
+const (
+	flagFeastRepoPath    = "FEAST_REPO_PATH"
+	flagFeastRepoConfig  = "FEAST_REPO_CONFIG"
+	flagFeastGrpcPort    = "FEAST_GRPC_PORT"
+	defaultFeastGrpcPort = "6566"
+	feastServerVersion   = "0.18.0"
+	defaultFeastHttpPort = "8081"
+)
+
+func main() {
+
+	repoPath := os.Getenv(flagFeastRepoPath)
+	repoConfig := os.Getenv(flagFeastRepoConfig)
+
+	if repoPath == "" && repoConfig == "" {
```

Comment:
We should also look for both being set (which shouldnt be the case)

Reply:
I think we want both even when repoConfig is in case user gives a relative registry path to repoPath?